File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1156.cs
using System;

class URI
{
    static void Main() {
        double a, b=2, c, soma=1.0;
        for(a=3;a<=39;a+=2){
            c=a/b;
            soma+=c;
            b*=2;
        }
    	Console.Write("{0:f2}\n", soma);

    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1156.cpp
#include <iostream>

using namespace std;

int main(){
    double a, b=2, c, soma=1;
    for(a=3;a<=39;a+=2){
        c=a/b;
        soma+=c;
        b*=2;
    }
    cout.precision(2);
    cout.setf(ios::fixed);
    cout << soma << endl;

    return 0;
}